Scalar.Common/ScalarPlatform.cs

```diff
@@ -114,6 +114,37 @@ public static void Register(ScalarPlatform platform)
             PhysicalFileSystem fileSystem,
             ITracer tracer);
 
+        public bool TryGetCredentialStore(ITracer tracer, PhysicalFileSystem fileSystem, out ICredentialStore credentialStore, out string errorMessage)
+        {
+            credentialStore = null;
+
+            string gitBinPath = ScalarPlatform.Instance.GitInstallation.GetInstalledGitBinPath();
+            if (string.IsNullOrEmpty(gitBinPath))
```

Comment:
In general, I like organizing things this way:

```cs
if (condition)
{
    // do easy thing
   return;
}

// Do more complicated thing
```

In this case, it would be

```cs
if (!string.IsNullOrEmpty(gitBinPath))
{
    // use existing git
    reutrn true;
}

// Find more complicated paths
```

Reply:
I've inverted the `if` to list the 'common' case (Git is found) first.